lib/player/db/postgres.go

```diff
@@ -98,7 +98,18 @@ func (p *PostgresTranslator) TranslateEvent(evt events.AuditEvent) *events.Sessi
 const lineBreak = "\r\n"
 
 func (p *PostgresTranslator) generateCommandPrint(metadata events.DatabaseMetadata, command string) string {
-	return lineBreak + fmt.Sprintf("%s=> %s", metadata.DatabaseName, command) + lineBreak
+	lead := metadata.DatabaseName + "=> "
+	leadSpacing := strings.Repeat(" ", len(lead))
+
+	var sb strings.Builder
+	commandLines := strings.Split(command, "\n")
```

Comment:
🤔 just thinking what if the original command uses `\r\n` instead of `\n`

Reply:
We'll replace the line break `\r\n` with another one.

That's why we [trim the spaces](https://pkg.go.dev/strings#TrimSpace), so we don't have duplicated line breaks or dangling spaces.